docs/scos/dev/front-end-development/202212.0/oryx/getting-started/oryx-environment.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Environment Variables"
```

Comment:
can we add other meta data here? especially the description

Reply:
added, always thought in it up to tech writer to define those